src/javascript/app_2/App/Components/Elements/Tabs/tabs_wrapper.jsx

```diff
@@ -0,0 +1,80 @@
+import React      from 'react';
+import PropTypes  from 'prop-types';
+import TabsItem   from './tabs_item.jsx';
+
+class TabsWrapper extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            sizes: {},
+        };
+        this.els = {};
+        this.getSizes = this.getSizes.bind(this);
```

Comment:
@zakibinary 
Since the `geSizes` method defined by arrow function, it doesn't need to bind it again in the constructor anymore.